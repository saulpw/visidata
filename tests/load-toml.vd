sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/freshwater-mammals.toml	o	
freshwater-mammals	muskrat	0	open-cell		zEnter	open sheet with copies of rows referenced in current cell
freshwater-mammals[['0']].muskrat	value	キvariants	open-cell		zEnter	open sheet with copies of rows referenced in current cell
freshwater-mammals[['0']].muskrat[variants].value	value	キround-tailed	open-cell		zEnter	open sheet with copies of rows referenced in current cell
freshwater-mammals	capybara	0	open-cell		zEnter	open sheet with copies of rows referenced in current cell
freshwater-mammals[['0']].capybara	value	キsightings	open-cell		zEnter	open sheet with copies of rows referenced in current cell
freshwater-mammals[['0']].capybara[sightings].value		0	open-row		Enter	open current row with sheet-specific dive
