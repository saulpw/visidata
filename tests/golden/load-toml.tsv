key	value
where	florida
when	2016-08-16
